docs/CHANGELOG.asciidoc

```diff
@@ -157,24 +159,30 @@ When you upgrade to 8.0.0, use the following API changes:
 
 * Use `service_tokens` instead of `service-tokens`.
 
-* `check-permissions` is no longer supported.
-
 * Use `/epm/packages/{packageName}/{version}` instead of `/epm/packages/{pkgkey}`.
 
-* Use `items[]` or `item` instead of `response[]` in the following:
-
+* Use `items[]` (or `item`) instead of `response[]` in:
```

Comment:
maybe remove the `(or item)` part from here to be clearer, the only use of `item` is highlighted below